thoughtis/cata-blocks
# Issue#122/flex grow control

### Related Issues
- Closes #122 

### What Was Accomplished
- Added a `cataBlocksFlexGrow` attribute with a number control that is only available if a block is fixed width

### How It Was Tested
- [x] Local parent theme
- [x] Local child theme
- [x] Staging child theme

### How To Test
- Flex grow number control appears for all blocks that are a direct child of a block with a `flex` layout and with the width set to `fixed`
- The `flex-grow` style for a block appears in the editor and on the frontend
- The `flex-grow` style is removed and the number control field is nulled if the block is switched to `fit` or `fill`
